recipes/allegro5/all/conanfile.py

```diff
@@ -0,0 +1,165 @@
+from conan.tools.microsoft import msvc_runtime_flag
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+class Allegro5Conan(ConanFile):
+    name = "allegro5"
+    license = ("ZLib", "BSD", "SDL", "Dejavu-fonts", "Creative Commons BY Attribution", "CBString")
```

Comment:
```suggestion
    license = ("ZLib", "BSD-3-Clause")
```
Reading https://github.com/liballeg/allegro5/blob/master/LICENSE.txt

I can found [Zlib](https://spdx.org/licenses/Zlib.html) and [BSD-3-Clause](https://spdx.org/licenses/BSD-3-Clause.html)

Examples and demos we not packaging, neither redistributing.

Reply:
Oh, right